internal/provider/schema_gitlab_application_settings.go

```diff
@@ -0,0 +1,3054 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func gitlabApplicationSettingsSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+
+		"admin_mode": {
+			Description: "Require administrators to enable Admin Mode by re-authenticating for administrative tasks.",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
+		},
+
+		"admin_notification_email": {
+			Description: "Deprecated: Use abuse_notification_email instead. If set, abuse reports are sent to this address. Abuse reports are always available in the Admin Area.",
```

Comment:
@timofurrer  - I'm not sure I feel extremely strong about this, but it seems weird to me to be building a new resource and have immediately deprecated fields. I know it's aligning to GitLab's API, but I can't think of a use-case for using this field in a new config other than using it for a very old version of GitLab. 

WDYT?

Reply:
I've removed all the deprecated fields from the resource.